SourcePointMetaApp/SourcePointMetaApp/Controllers/AddWebsiteViewController/AddWebsiteViewController.swift

```diff
@@ -64,17 +89,17 @@ class AddWebsiteViewController: BaseViewController,TargetingParamCellDelegate, U
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        accountIDTextFieldOutlet.delegate = self
-        websiteTextFieldOutlet.delegate = self
-        targetingParamTableview.tableFooterView = UIView(frame: .zero)
-        setTableViewHidden()
-        
+//        accountIDTextFieldOutlet.delegate = self
+//        websiteTextFieldOutlet.delegate = self
+//        targetingParamTableview.tableFooterView = UIView(frame: .zero)
+//        setTableViewHidden()
+                
```

Comment:
@vilasma do we need the commented out code? Shall we remove it?

Reply:
To keep the changes minimum in  PR I have commented that code but its not there in  final version of app

Reply:
In other PR you can may seen that lots of database related code is commented out but thats because I want to keep minimum changes in single commit.